shared/fs/row.js

```diff
@@ -3,44 +3,45 @@ import * as Types from '../constants/types/fs'
 import {connect, type TypedState, type Dispatch} from '../util/container'
 import {isMobile} from '../constants/platform'
 import {navigateAppend} from '../actions/route-tree'
+import type {IconType} from '../common-adapters/icon'
 
 type OwnProps = {
   path: Types.Path,
 }
 
-type StateProps = {
-  type: Types.PathType,
+type DispatchProps = {
+  _onOpen: (type: Types.PathType, path: Types.Path) => void,
 }
 
-type DispatchProps = {
-  _onViewFolder: Types.Path => void,
-  _onViewFile: Types.Path => void,
+const mapStateToProps = (state: TypedState, {path}: OwnProps) => {
+  const pathItem = state.fs.pathItems.get(path)
```

Comment:
the actual issue you had. getIn was getting confused and its not clear to me why. using flow check --trackes=100 revealed this

Reply:
I did that and it didn't help, just gave me the same message. What about that command revealed it?

Reply:
Coverage also shows this. The getin was red

Reply:
I tried switching it to `get` and it was also confused. Not sure why it works for you.

Reply:
Although I guess I didn't try moving it _after_ I created the typing for `PathType`.